notebooks/official/automl/sdk_automl_text_sentiment_analysis_online.ipynb

```diff
@@ -0,0 +1,951 @@
+{
```

Comment:
<p>Don't hardcode, see notebook_template.ipynb.</p><p> </p><p>Should be:</p><p> </p><pre class="ql-syntax" spellcheck="false">REGION = "[your-region]"  # @param {type:"string"}
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>